src/chordChart.ts

```diff
@@ -323,6 +323,8 @@ module powerbi.extensibility.visual {
                 dataMatrix.push([]);
                 toolTipData.push([]);
 
+                let localManager = host.createLocalizationManager();
```

Comment:
Does `createLocalizationManager` always return the same instance of LocalizationManager?
If it creates a new one for each update we should call `createLocalizationManager` once in the constructor and keep it as `this.localizationManager`.